sonargraphintegration.properties

```diff
@@ -4,20 +4,26 @@ category=External Analyzers
 description=Provides support for Sonargraph 8+'s architecture governance features, accompanied by metrics about cyclic dependencies and other structural aspects.
 homepageUrl=https://github.com/sonargraph/sonar-sonargraph-integration
 devVersion=
-publicVersions=5.2.0
-archivedVersions=1.1.2,1.1.3,1.1.4,2.0.1,2.1.5,2.1.7,3.1.0,4.0.0,4.1.1,4.1.2,5.0.0,5.1.0
+publicVersions=6.0.0
+archivedVersions=1.1.2,1.1.3,1.1.4,2.0.1,2.1.5,2.1.7,3.1.0,4.0.0,4.1.1,4.1.2,5.0.0,5.1.0,5.2.0
 privateVersions=
 
 defaults.mavenGroupId=com.hello2morrow
 defaults.mavenArtifactId=sonar-sonargraph-integration
 
+6.0.0.description=Removal of Sonargraph rules 'Script-Based' and 'Plugin-Based' in favor of individual rules per script and plugin issue.
+6.0.0.sqVersions=[7.9,LATEST]
+6.0.0.date=2020-11-25
+6.0.0.changelogUrl=https://github.com/sonargraph/sonar-sonargraph-integration/releases/download/Release-6.0.0/release-notes.txt
+6.0.0.downloadUrl=https://github.com/sonargraph/sonar-sonargraph-integration/releases/download/Release-6.0.0/sonar-sonargraph-integration-6.0.0.jar
+
+#====================
 5.2.0.description=Compatibility with Sonargraph 10.4.
-5.2.0.sqVersions=[7.9,LATEST]
+5.2.0.sqVersions=[7.9,8.5]
```

Comment:
This needs to end with 8.5.* or 8.5.1.
Note that 8.6 is currently expected on Monday, so... FYI.